modules/custom/az_paragraphs/az_paragraphs.install

```diff
@@ -83,3 +83,10 @@ function az_paragraphs_update_920601(&$sandbox) {
   $sandbox['#finished'] = ($sandbox['total'] === '0') ? 1 : ($sandbox['current'] / $sandbox['total']);
   return t('Paragraph behavior settings fixed on %count paragraphs.', ['%count' => $sandbox['updated_count']]);
 }
+
+/**
+ * Enable anchor_link module for CKEditor5 by default.
+ */
+function az_quickstart_update_1021002() {
```

Comment:
Update number may need to change depending on whether we decide to include this in a patch release and/or backport.